src/ostree/ot-builtin-summary.c

```diff
@@ -40,6 +40,63 @@ static GOptionEntry options[] = {
   { NULL }
 };
 
+/* Take arguments of the form [key=value] or [key, value] out of @argv and put
+ * them into an a{sv} variant. The value arguments must be parsable using
+ * g_variant_parse(). */
+static GVariant *
+argv_to_additional_metadata (int      *argc,
+                             char   ***argv,
+                             GError  **error)
+{
+  g_autoptr(GVariantBuilder) builder = NULL;
+  int i;
+
+  builder = g_variant_builder_new (G_VARIANT_TYPE_VARDICT);
+
+  for (i = 1; i < *argc; i++)
```

Comment:
For `i`? The declaration is deliberately in a higher scope, since there’s a second loop which counts down afterwards.